packages/fiori/src/NotificationList.ts

```diff
@@ -36,6 +36,23 @@ type NotificationItemCloseEventDetail = NotificationItemEventDetail;
  * The `ui5-notification-list` web component represents
  * a container for `ui5-li-notification-group` and `ui5-li-notification`.
  *
+ * ### Keyboard Handling
```

Comment:
Notification List component doesn't have a page in the website and this information is not visible in the documentation.